src/Data/Tree/AVL/Store/Pure.hs

```diff
@@ -1,8 +1,8 @@
 -- | Pure in-memory storage for AVL tree.
 module Data.Tree.AVL.Store.Pure
     ( -- * Carrier monad
-      Store
-    , State
+      PureStore
```

Comment:
Oh, nice, you've just changed public API. And this module is supposed to be used qualified, like `Pure.Store, Pure.State`.

Reply:
This module is actually used in my PR for DSCP-193.

Because of all that, I propose to roll this renaming back.